### PR TITLE
Incompleteness

### PR DESCRIPTION
**Pull request by williums**
This should be ok now, nothing too big.
- [x] Expense items have a flag to indicate incompleteness
- [x] Incomplete expenses have an icon in the listview
- [x] Implement a confirmation dialog for submitting expense claims. An additional message will be displayed if there are any expense items marked as incomplete.

### CODE REVIEW

**Comment by indragiek** [comment]
Sorry for the delay in reviewing, will get to this either tonight or tomorrow morning.

**Comment by indragiek** [comment]
This looks fine to me! @CMPUT301W15T09/owners any other comments?

File: Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java [hunk 1]
```diff
@@ -424,6 +423,34 @@ public boolean onOptionsItemSelected(MenuItem item) {
 		}
 	}
```
Comment (mrthegreensky) [inline Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java:415]: Add java docs

File: Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java [hunk 2]
```diff
@@ -424,6 +423,34 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+		
+		openDialog.setNegativeButton(R.string.action_cancel, new DialogInterface.OnClickListener() {
+			@Override
+			public void onClick(DialogInterface dialog, int which) {
+				dialog.dismiss();
+			}
+		});
+		openDialog.show();
+	}
+
```
Comment (mrthegreensky) [inline Team9Project/src/com/indragie/cmput301as1/ExpenseClaimDetailActivity.java:447]: Add java docs

**Comment by indragiek** [comment]
Looks like all of the comments have been addressed. @mrthegreensky anything else or is this good to merge?

**Comment by mrthegreensky** [comment]
@indragiek I think it looks good.

**Pull request merged**